website/client/src/components/shops/gemPrice.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div
+    class="gem-price-div"
+    :class="{'background': withBackground}"
+  >
+    <div
+      :class="'mr-2 svg-icon gem icon-' +iconSize"
```

Comment:
Is this a new recommended pattern? I'm used to us doing \``mr-2 svg-icon gem icon-${iconSize}`\`

Reply:
nah no new pattern ha - also I didn't even know we could use the template string syntax in HTML as well  - gonna try that